packages/java-edition/src/binder/index.ts

```diff
@@ -170,52 +161,40 @@ export function* getRels(
 }
 
 export function dissectUri(uri: string, ctx: UriBinderContext) {
-	const regex =
-		/^data\/([^\/]+)\/((?:tags\/)?(?:worldgen\/)?[a-z_]+)\/(.*)(\.(?:mcfunction|json|nbt))$/
-
 	const rels = getRels(uri, ctx.roots)
-	const { customResources } = ctx.config.env
+	const release = ctx.project['loadedVersion'] as ReleaseVersion | undefined
```

Comment:
why do we access `loadedVersion` with a string index here instead of just normal property access?

```ts
const release = ctx.project.loadedVersion // ...
```

Reply:
We do that everywhere else in the project as well. I'm not sure 🤷